src/core/src/op/divide.cpp

```diff
@@ -72,7 +73,7 @@ ov::Tensor equality_mask(const ov::Tensor& tensor, const shared_ptr<op::Constant
 }
 
 ov::Tensor or_tensor(const ov::Tensor& lhs, const ov::Tensor& rhs) {
-    auto outs = ov::TensorVector{{lhs.get_element_type(), Shape{0}}};
+    auto outs = ov::TensorVector{{lhs.get_element_type(), ov::util::make_dynamic_shape()}};
```

Comment:
Why do we need to use here the new WA? I would like to have it only as some internal temporary thing in order to connect two APIs.

Reply:
Looks is not required here. Changed.